### DISCUSSION TITLE
next build error

### DISCUSSION THREAD

**Discussion by DinhThuongNguyen** [vercel/next.js#30688] [Help]
### What version of Next.js are you using?

11.1.2

### What version of Node.js are you using?

16.10.0

### What browser are you using?

chrome

### What operating system are you using?

win 10

### How are you deploying your application?

next build

### Describe the Bug

I use getStaicProps and getStaticPaths in dynamic routes, when using yarn build it always gives error

### Expected Behavior

If i change getstaticProps(and comment getStaticPaths ) to getServerSideProps then build doesn't give error

### To Reproduce

[https://postimg.cc/7bwNVL1s](https://postimg.cc/7bwNVL1s)
[https://postimg.cc/21bd6Lm6](https://postimg.cc/21bd6Lm6)
[https://postimg.cc/5Y3LGw5P](https://postimg.cc/5Y3LGw5P)
[https://github.com/DinhThuongNguyen/code-web-manga/blob/main/pages/truyen-tranh/%5BnameTruyen%5D/%5BchapTruyen%5D.js](https://github.com/DinhThuongNguyen/code-web-manga/blob/main/pages/truyen-tranh/%5BnameTruyen%5D/%5BchapTruyen%5D.js)

**Comment by timneutkens** [comment]
This seems to be a problem in your application, the `getStaticProps` does not resolve within 60 seconds so we retry 3 times before failing, do you get a response when using getServerSideProps? Maybe the credentials for your database are not provided during build?
